<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_Team</class>
 <widget class="QMainWindow" name="Gui_Team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/teams_48</normaloff>:/icons/teams_48</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="teamList">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>128</height>
       </size>
      </property>
      <property name="toolTip">
       <string>List of all participating teams</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="addEditWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetClosable</set>
      </property>
      <property name="windowTitle">
       <string>Add/Edit Team</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="addEditLayout" columnstretch="1,1,1,1">
          <item row="0" column="0">
           <widget class="QLabel" name="teamNameLabel">
            <property name="text">
             <string>Team name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="teamNameEdit">
            <property name="toolTip">
             <string>Type in team's name</string>
            </property>
            <property name="placeholderText">
             <string>Type team name here</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QTimeEdit" name="finishTimeEdit">
            <property name="toolTip">
             <string>Change team's finish time</string>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::HourSection</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="finishTimeLabel">
            <property name="text">
             <string>Finish time</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="doneButton">
            <property name="toolTip">
             <string>Click when done editing team</string>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/ok_16</normaloff>:/icons/ok_16</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QPushButton" name="cancelButton">
            <property name="toolTip">
             <string>Click to abort editing</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/cancel_16</normaloff>:/icons/cancel_16</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="teamMembersLabel">
            <property name="text">
             <string>Members</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="teamMembersEdit">
            <property name="toolTip">
             <string>Change number of team members</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="reviewerLabel">
            <property name="text">
             <string>Reviewer</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="reviewerButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/reviewer_edit_16</normaloff>:/icons/reviewer_edit_16</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="reviewerEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="toolTip">
         <string>Click to close team dialog</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/close_16</normaloff>:/icons/close_16</iconset>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/team_add_16</normaloff>:/icons/team_add_16</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/team_remove_16</normaloff>:/icons/team_remove_16</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/team_edit_16</normaloff>:/icons/team_edit_16</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
